src/configuration/validators/neovimValidator.ts

```diff
@@ -30,7 +29,7 @@ export class NeovimValidator implements IConfigurationValidator {
             });
           }
         }
-        await promisify(execFile)(config.neovimPath, ['--version']);
+        execFileSync(config.neovimPath, ['--version']);
```

Comment:
Updated this to `execFileSync` because we're essentially using it synchronously and it's much easier to mock.